src/server/apsystem.ts

```diff
@@ -93,6 +93,11 @@ export default class ActivityPubSystem {
   async verifySignedRequest (request: FastifyRequest, fromActor?: string): Promise<string> {
     // TODO: Fetch and verify Digest header
     const { url, method, headers } = request
+
+    if (headers.signature === undefined) {
+      return new Promise((resolve, reject) => resolve(""))
```

Comment:
```suggestion
      return ""
```

This is an async function; it always returns a Promise, so there's no need to wrap it in a Promise. (This still works because Promises that resolve Promises get "flattened")

If this wasn't the case (say, this was a normal function that returned a Promise), there's a shorter way to do the same thing: `return Promise.resolve("")`

Reply:
aah thanks! what about using an empty string vs adding a constant like `"nonexistent@act.or"`?